src/adapter/scriptSkipper.ts

```diff
@@ -27,24 +44,18 @@ export class ScriptSkipper {
   }
 
   private _preprocessNodeInternals(): void {
-    const nodeInternalRegex = /^<node_internals>[\/|\\\\](.*)$/;
-    const skipAllNodeInternalsRegex = /^<node_internals>[\/|\\\\]\\*\\*[\/|\\\\]\\*.js/;
-
-    const nodeInternalPatterns = this._userSkipPatterns
-      .filter(pattern => pattern.includes('<node_internals>'))
-      .map(nodeInternal => {
-        nodeInternal = nodeInternal.trim();
-        if (skipAllNodeInternalsRegex.test(nodeInternal)) {
-          // check if all node internals are skipped
-          this.skipAllNodeInternals = true;
-        }
-
-        const match = nodeInternalRegex.exec(nodeInternal);
-        return match ? match[1] : nodeInternal;
-      });
-
-    if (!this.skipAllNodeInternals && nodeInternalPatterns.length > 0) {
-      this._nodeInternalsRegex = this._createRegexString(nodeInternalPatterns);
+    const nodeInternalRegex = /^<node_internals>[\/\\](.*)$/;
+
+    const nodeInternalPatterns = this._userSkipPatterns!.filter(pattern =>
```

Comment:
`_userSkipPatterns` is just used to initialize some other properties and isn't referenced after the constructor is done, so it shouldn't be a property, just pass it to these initialize methods as an argument. It will be more clear how it's used.